Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/README.md

```diff
@@ -20,29 +20,34 @@ For detailed instructions about setting up authentication, see: [AWS Integration
-   | Role Session Duration | The maximum length of each session in seconds. Default: 900 seconds. The integration will have the permissions assigned only when the session is initiated and for the defined duration. | False |
-   | Access Key | The access key ID used for authentication, that was configured during IAM user configuration. If this is used, Role ARN is not required. | False |
-   | Secret Key | The secret key used for authentication, that was configured during IAM user configuration. If this is used, Role ARN is not required. | False |
-   | Timeout | The secret key used for authentication, that was configured during IAM user configuration. If this is used, Role ARN is not required. | False |
-   | Retries | The maximum number of retry attempts when connection or throttling errors are encountered. Set to 0 to disable retries. The default value is 5 and the limit is 10. Note: Increasing the number of retries will increase the execution time. | False |
-   | Trust any certificate (not secure) |  | False |
-   | Use system proxy settings |  | False | 
-   | Security Hub Severity level | Severity level of fetched incidents. Could be Low, Medium and High. For example, if you set the severity level to Medium, only findings with severity level Medium or High will be fetched. | False |
-   | Additional Filters | A comma-separated list of additional incident filters in the form 'name=name1,value=value1,comparison=comparison1 | False |
-   | Change findings workflow to 'NOTIFIED' | Notify the resource owner about the security issue. You can use this status when you are not the resource owner, and you need intervention from the resource owner in order to resolve a security issue. | False |
-6. Click __Test__ to validate the URLs, token, and connection.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description**                                                                                                                                                                                                                                                                                                          | **Required** |
+    |--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| --- | --- |
+    | Role Arn |                                                                                                                                                                                                                                                                                                                          | False |
```

Comment:
Why were the descriptions removed for the following parameters?
- Role Arn
- Role Session Name
- AWS Default Region
- Role Session Duration
- Access Key
- Secret Key